account/tests.py

```diff
@@ -183,4 +184,69 @@ def test_sign_in_invalid_email(self):
             {
                 'message': 'INVALID_EMAIL'
             }
-        )
\ No newline at end of file
+        )
+
+class FacebookSignInViewTest(TestCase):
+    def setUp(self):
+        SocialLoginType.objects.create(name = 'facebook')
+        User.objects.create(
+            name                = 'testname',
+            email               = 'test@gmail.com',
+            password            = '',
+            image_url           = '',
+            social_login_id     = '3187176975214272',
+            social_login_type   = SocialLoginType.objects.get(name = 'facebook')
+        )
+        
+    def tearDown(self):
+        User.objects.all().delete()
+        SocialLoginType.objects.all().delete()
+
+    @patch('account.views.requests')
+    def test_facebook_signin_view_exists(self, mocked_request):
+        class FakeResponse:
+            def json(self):
+                return {
+                    'id'    : '3187176975214272',
+                    'name'  : 'testname',
+                }
+
+        mocked_request.get  = MagicMock(return_value = FakeResponse())
+        client              = Client()
+        header              = {'Authorization' : 'fake.access_token'}
```

Comment:
HTTP_Authorization 키로 보내줘야 view 로직에서 request.headers.get('Authorization') 로 get 할 수 있습니다. 확인 부탁드립니다.

```suggestion
        header              = {'HTTP_Authorization' : 'fake.access_token'}
```